lib/src/main/java/io/github/taz03/jia/InstagramClient.java

```diff
@@ -49,20 +56,40 @@ public final class InstagramClient {
      * @param password The instagram password corresponding to the username provided
      */
     public InstagramClient(String username, String password) {
-        this(username, password, null);
+        this.username = username;
+        this.password = password;
+    }
+
+    public InstagramClient(String user,String pass,String id,String agent){
+        this.username = user;
+        this.password = pass;
+        this.userAgent = agent;
+        this.androidId = id;
+    }
+
+    private void configureDeviceIds() throws MalformedURLException, IOException {
+        var id = UUID.randomUUID().toString().replaceAll("-", "").toUpperCase();
+        this.androidId += "android-" + id.substring(id.length() - 16);
+        
+        var random = new Random().nextInt(1783);
+        var scanner = new Scanner(new URL("https://raw.githubusercontent.com/Taz03/JIA/main/user-agents.txt").openStream());
```

Comment:
use strict types and not `var`

Reply:
Any specific reason ?

Reply:
readability, this is not standard among java dev as well

Reply:
but I guess `var` makes it looks more good. It's a new fashion in programming (inspired by kotlin).
You can change them if you want but it looks good and more readable right now, it's just that you need to identify the class by the return value of the method

Reply:
> but I guess var makes it looks more good. It's a new fashion in programming (inspired by kotlin).

ur really pulling this out of no where

Reply:
I will rather say, I am actually using **new java feature**